RocketChat/Rocket.Chat
# [FIX] Unintended "Reply on thread" button showing on thread's first message

## Proposed changes (including videos or screenshots)


Referent to TC-128

#### Before:
![image](https://user-images.githubusercontent.com/47800334/211622867-5d24a4c9-172b-4b36-aac9-cea1af3f925a.png)

#### After:
![image](https://user-images.githubusercontent.com/47800334/211622729-14e8c552-615c-44e3-b5ae-8e815909e5ac.png)



## Issue(s)


## Steps to test or reproduce

Open a new channel, then:
1. Send any message
2. Click "Reply in thread" button and send a message in thread to that message
3. You'll see that the button "Reply in thread" won't show anymore for the top message, avoiding "thread nesting" and limiting it to only 1 thread, as it should be
4. Also any other messages sent in that thread won't be allowed to create another inside it

## Further comments

As flagged in comment on that issue, seems that there was a discussion whether the thread should open again or have a new created one when clicking on such unintended button. That said, it's conclusive to say that there was no point on opening a new thread for the top message when you're already on one, same for opening new threads for further messages.
